__tests__/app.test.js

```diff
@@ -16,3 +23,31 @@ describe.only("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: responds with an array of topic objects, each containing slug and description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toBeInstanceOf(Array);
+
+        body.topics.forEach((topic) => {
```

Comment:
If using a for each you should assert that this array has a length. With an empty array any expect statements inside will be skipped.